### PR TITLE
CMR-8991 fixes collections conversion

### PR DESCRIPTION
**Pull request by jaybarra**
Fixes a case where a single collection would not have the IDs properly converted.
Adds architectural decision logs.

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -54,8 +54,8 @@ export const stacContext = (req: Request) => {
     id: isCloudStac ? "CLOUDSTAC" : "STAC",
     root,
     stacRoot: `${root}/${stac}`,
-    path: `${root}${path}`,
```
Comment (jaybarra) [inline src/utils/index.ts:57]: STAC browser included a trailing `/` in the URLS, which broke many of the links URLs for each item and collection.

File: adr-log/adr-20221201-graphql-backend.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Using direct calls to CMR make it necessary to combine multiple call results to produce STAC items. There is also no mechanism to call for only what fields are needed. This results in extra overhead, both in terms of number of calls needed and extra data being returned that was ultimately discarded.
+
+## Decision
+
+Replace the logic in CMR-STAC to query GraphQL for data instead of querying directly to CMR.
+
+## Consequences
+
+### Pros
++ Constructing STAC responses requires far fewer calls to generate the same responses.
```
Comment (jceaser) [inline adr-log/adr-20221201-graphql-backend.md:24]: suggestion, use either * (preferably *) or + not both.

File: adr-log/adr-20230323-entry-id-title-identifiers.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## Consequences
+
+There is ambiguousness regarding collections when the shortName contains a ".v" separator. The `.v` must be converted to an underscore to be transformed into a valid CMR `entry_id`. This transformation is only required for navigating directly to a collection or item
+
+### Bookmarked Collections
+
+Bookmarked collection pages will need to be updated to change the ".v" to "_":
+
+- INCORRECT `/stac/provider/collections/myCollection.v1`
```
Comment (jceaser) [inline adr-log/adr-20230323-entry-id-title-identifiers.md:30]: mixing list markers, ether * or -

**Review by jceaser — APPROVED** [review]
the single quote comment is the most important, the others are suggestions.

File: adr-log/adr-20190926-initial-design.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+* [Decision](#decision)
+* [Consequences](#consequences)
+
+## Status
+
+__Accepted__
+
+## Context
+
+There is a community desire to be able to use the Spatio-Temporal Asset Catalog specification to browse data holdings within NASA' Common Metadata Repository.
```
[the remark this answers is not in the document]
Comment (jceaser) [reply adr-log/adr-20190926-initial-design.md:15]: I am wearing a different pair of glasses today, maybe I just did not see it.  8)

**Review by dpesall — APPROVED** [review]

**Review by eudoroolivares2016 — APPROVED** [review]

**Pull request merged**